tests/lib/config/config-file.js

```diff
@@ -127,6 +128,28 @@ function createStubModuleResolver(mapping) {
     };
 }
 
+/**
+ * Overrides the native module resolver to resolve with the given mappings.
+ * @param {Object<string,string>} mapping A mapping of module name to path.
+ * @returns {void}
+ * @private
+ */
+function overrideNativeResolve(mapping) {
+    const originalFindPath = Module._findPath; // eslint-disable-line no-underscore-dangle
+
+    beforeEach(() => {
+        Module._findPath = function(request, paths, isMain) { // eslint-disable-line no-underscore-dangle
```

Comment:
This feels a little risky to me. If someone writes `overrideNativeResolve(mapping)` more than once in a given describe block, then we could have two levels of overriding of `Module._findPath`.

I think it would make more sense to just override `Module._findPath` unconditionally to try to look up within the mappings and fallback to the original `Module._findPath` as a fallback, and then `overrideNativeResolve` would just set up and remove the mappings.